subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/TaskErrorExecutionIntegrationTest.groovy

```diff
@@ -182,6 +183,9 @@ class TaskErrorExecutionIntegrationTest extends AbstractIntegrationSpec implemen
         fails "someTest"
 
         then:
+        verifyAll(receivedProblem(0)) {
+            fqid == 'task-selection:no-matches'
+        }
```

Comment:
🤔 
Should we also test other parts of the problem?
like label?

Reply:
The contextual label for sure